host/app/components/card-editor.gts

```diff
@@ -79,12 +80,15 @@ export default class Preview extends Component<Signature> {
   rendered = render(this, () => this.card, () => this.format)
   @tracked
   initialCardData: CardJSON | undefined;
+  private interval: ReturnType<typeof setInterval>;
 
   constructor(owner: unknown, args: Signature['Args']) {
     super(owner, args);
     if (this.args.card.type === 'existing') {
       taskFor(this.loadData).perform(this.args.card.url);
     }
+    this.interval = setInterval(() => taskFor(this.loadData).perform((this.args.card as any).url ?? ''), 1000);
```

Comment:
@burieberry is there a reason why we set this to empty string if the url is undefined? seems like a better approach is to just accept an undefined value in the loadData() task

Reply:
Realized the `as any` already resolved the typescript issue I was trying to circumvent